monai/apps/detection/networks/retinanet_detector.py

```
@@ -519,7 +518,7 @@ def forward(
         else:
             if self.inferer is None:
                 raise ValueError(
-                    "`self.inferer` is not defined." "Please refer to function self.set_sliding_window_inferer(*)."
+                    "`self.inferer` is not defined.Please refer to function self.set_sliding_window_inferer(*)."
                 )
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)